Dockerfile

```diff
@@ -77,9 +77,17 @@ COPY presto-raptor-legacy /build/presto-raptor-legacy
 COPY presto-password-authenticators /build/presto-password-authenticators
 COPY src /build/src
 COPY pom.xml /build/pom.xml
+#build debug info
+RUN cat /proc/meminfo
+
+# increase memory available to Maven
+RUN export MAVEN_OPTS='-Xmx512m -XX:MaxPermSize=128m'
+
+# load Maven cache as its own layer
+RUN cd /build && mvn de.qaware.maven:go-offline-maven-plugin:resolve-dependencies -DdownloadSources -DdownloadJavadoc -Dmaven.repo.local=.m2/repository
```

Comment:
Is there any reason we can't just specify `WORKDIR /build` and avoid having to change to the build directory everytime?

Reply:
yeah I saw that and wondered. I think it'd be fine. Will add.